virttest/utils_test/qemu/__init__.py

```diff
@@ -735,6 +753,8 @@ def check_memory(self, vm=None, wait_time=0):
         #  let system perceive it.
         time.sleep(wait_time)
         guest_mem_size = super(MemoryHotplugTest, self).get_guest_total_mem(vm)
+        crashkernel_size = super(MemoryHotplugTest, self).get_guest_crashkernel_size(vm)
```

Comment:
`check_memory` can get Windows/Linux memory, but crashkernel only for Linux. So when the guest is Windows guest, you should skip it.

Reply:
thanks a lot, has been corrected.